scripts/unocommands.py

```diff
@@ -34,10 +36,12 @@ def usageAndExit():
     {program} --translate /path/to/online /path/to/translations
 
 """
-    print(message.format(program = sys.argv[0]))
+    print(message.format(program=sys.argv[0]))
     exit(1)
 
 # Extract uno commands name from lines like "  'Command1', 'Command2',"
+
```

Comment:
These two new blank lines separate the function description above from its function below. Please remove these new empty lines. And move the comment above into the function as in the example below:

```python
def quadratic(a, b, c, x):
    """Use the quadratic formula"""
    x_1 = (- b+(b**2-4*a*c)**(1/2)) / (2*a)
    x_2 = (- b-(b**2-4*a*c)**(1/2)) / (2*a)

    return x_1, x_2
```